CRM/Upgrade/Form.php

```diff
@@ -31,15 +31,6 @@ class CRM_Upgrade_Form extends CRM_Core_Form {
    */
   const MINIMUM_UPGRADABLE_VERSION = '4.2.9';
 
-  /**
-   * Minimum php version required to run (equal to or lower than the minimum install version)
-   *
-   * As of Civi 5.16, using PHP 5.x will lead to a hard crash during bootstrap.
-   *
-   * Tip: Keep in sync with composer.json ("config => platform => php")
-   */
-  const MINIMUM_PHP_VERSION = '7.1.0';
-
```

Comment:
For removing `MINIMUM_PHP_VERSION`, we'd probably want to sequence the merges s.t. references are first removed from `civicrm-{backdrop,wordpress,drupal}.git` and lastly `civicrm-core.git`.

Or if you go the other way (replacing `MIN_INSTALL_PHP_VER`), then I think it only needs one patch.

Reply:
I agree about merging the other repos first.  I think it belongs in `CRM_Upgrade_Incremental_General` both because the other PHP versions are defined there and because that class is intended as a general utility class rather than a form class.